<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udg::QModalitiesSelectorGroupBoxBase</class>
 <widget class="QGroupBox" name="udg::QModalitiesSelectorGroupBoxBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Modality</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="m_allCheckBox">
     <property name="text">
      <string>All</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="m_CRCheckBox">
     <property name="text">
      <string>CR</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="m_ESCheckBox">
     <property name="text">
      <string>ES</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="m_NMCheckBox">
     <property name="text">
      <string>NM</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="m_RFCheckBox">
     <property name="text">
      <string>RF</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="m_XACheckBox">
     <property name="text">
      <string>XA</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="m_CTCheckBox">
     <property name="text">
      <string>CT</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="m_MGCheckBox">
     <property name="text">
      <string>MG</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="m_OPCheckBox">
     <property name="text">
      <string>OP</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="m_SCCheckBox">
     <property name="text">
      <string>SC</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="m_XCCheckBox">
     <property name="text">
      <string>XC</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="m_DXCheckBox">
     <property name="text">
      <string>DX</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="m_MRCheckBox">
     <property name="text">
      <string>MR</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="m_PTCheckBox">
     <property name="text">
      <string>PT</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QCheckBox" name="m_USCheckBox">
     <property name="text">
      <string>US</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QCheckBox" name="m_otherCheckBox">
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLineEdit" name="m_otherLineEdit">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
